docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,645 @@
+= Migration Tutorial: From APPUiO Public to APPUiO Cloud
+
+This document provides an overview of all the tasks required to move your applications from APPUiO Cloud to APPUiO Public.
+
+== Main Differences
+
+Before starting the migration of your applications from APPUiO Public to APPUiO Cloud, please check the xref:differences-to-public.adoc[document explaining all the major differences] between both platforms.
+
+=== Timezone
+
+As a reminder, please keep in mind that in APPUiO Public the timezone was *Europe/Zurich*; however, in APPUiO Cloud (based on OpenShift 4) the control plane runs on *UTC* instead.
+
+Example of getting the current time on a control plane:
+
+[source,bash]
+--
+sh-4.4# date
+Mon Oct 25 14:41:20 UTC 2021
+--
+
+For all time and date information in your manifests and commands, please change it to conform to UTC, or better yet, beginning with OpenShift 4.9 (based on Kubernetes v1.21) you should define the timezone in a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[Kubernetes CronJob].
+
+== DNS
+
+Customers transferring their domains from APPUiO Public to APPUiO Cloud should be aware of the following:
+
+. Lowering your TTLs (Time To Live) 24 hours ahead of time will reduce the amount of time that your change takes to propagate. You can limit downtime by lowering that value, so the caching servers will check back more frequently. Typically, 300 seconds (5 minutes) instead of the default of 14400 (4 hours) is a good idea.
++
+[source,bash]
```

Comment:
I'm not sure how much value this example brings to this stage, I'd just remove it.

Reply:
Removed